### PR TITLE
v1.3 fixes

### PR DESCRIPTION
**Pull request by lovincyrus**

### CODE REVIEW

**Review by nassdonald — DISMISSED** [review]
Some more minor bugs to fix:
- [x] The north-east arrow renders as an emoji in Safari
image
- [x] The graphics in the testnet explorers section need to be `z-index -1` so they don't prevent interaction with the links in the above sections.
- [x] The explorer cards should have `hover-raise()` to elevate them on interaction
- [x] The width of the article on the testnet page should be reduced so the north-east arrow isn't too far from the text.
- [x] The homepage hero seems to be missing horizontal padding. Just needs a `tm-section-container` class on the `section-container` element, I think.
image
- [x] "Learn more" link on the homepage second section (intro) should link to the /resources page.

**Comment by nassdonald** [comment]
Let's add the additional changes to the Roadmap section as outlined in [Asana task](https://app.asana.com/0/1191880207702137/1199533373712432/f). 🙏

File: components/BugBountyBanner.vue [hunk 1]
```diff
@@ -43,8 +45,8 @@
 </template>
 
 <style lang="stylus" scoped>
-.section
-  overflow visible
+// .section
```
Comment (nassdonald) [inline components/BugBountyBanner.vue:48]: Probably best to remove comments as a matter of habit.

File: components/SectionRoadmap.vue [hunk 2]
```diff
@@ -6,23 +6,24 @@
...
         <div class="section-milestones">
           <div class="section-milestones__title tm-rf3 tm-bold tm-lh-title">
-            Milestones
+            Releases
           </div>
           <div class="section-milestones__cta">
             <tm-button
               to-link="external"
-              href="https://github.com/orgs/cosmosdevs/projects/1"
+              href="https://github.com/cosmos/cosmos-sdk/releases/tag/v0.40.0"
```
Comment (nassdonald) [inline components/SectionRoadmap.vue:19]: This should stay the same URL – but maybe change the link text to "View project ↗️". The URL for the Cosmos SDK item on the right was meant to be updated with this release URL.

File: components/SectionRoadmap.vue [hunk 3]
```diff
@@ -105,15 +107,15 @@ export default {
...
-  computed: {
-    progressTotal() {
-      const progressSum = this.milestoneList
-        .map((i) => Math.floor(i.progress))
-        .reduce((a, b) => a + b, 0)
-      const percentage = ((progressSum / 300) * 100).toFixed(0)
-      return percentage
-    },
-  },
+  // computed: {
```
Comment (nassdonald) [inline components/SectionRoadmap.vue:110]: Is there a reason for keeping commented code?

**Review by nassdonald — CHANGES_REQUESTED** [review]
Just a minor tweak to the URL.

File: components/SectionRoadmap.vue (hunk 3 above)
Comment (lovincyrus) [reply components/SectionRoadmap.vue:110]: If we need it later down the road. I can see this being reused in the future. It's also part of the reason why I kept the old components (SectionNavOld, SectionTestnetsOld).

**Review by nassdonald — CHANGES_REQUESTED** [review]
Just a few more bugs and small improvements to make:
- [x] GitHub links on testnet pages are all broken - not linking to an external URL.
- [x] The copy in the main CTA button in the hero should be "Prepare" instead of "Join testnets"
- [x] Add a second CTA button in the hero with the copy "Vote", linking to [this URL](https://www.mintscan.io/cosmos/proposals/35), and with this styling:
```
background: var(--gray-trans-300); // new variable needed: rgba(0, 4, 56, 0.738)
color: var(--white);
backdrop-filter: blur(3rem);
```
- [ ] The "Testnets" primary nav item, footer link, and any back links from the testnet detail pages should be updated to "Prepare".
- [ ] The proposal CTA panel should link to https://www.mintscan.io/cosmos/proposals/35 now that it's live. Also " · Draft" should be removed from the overline.
- [ ] The CTA panels should be reordered to have the proposal CTA at the top.
- [ ] The "Join the testnets -->" title of the CTA panel should be changed to "Prepare for Stargate -->".

**Comment by lovincyrus** [comment]
> * The "Testnets" primary nav item, footer link, and any back links from the testnet detail pages should be updated to "Prepare".

Are we only changing the copy from Testnets → Prepare? Do I need to change the URLs as well? From /testnets → /prepare @nassdonald

**Comment by nassdonald** [comment]
Nah, I think the URL should be fine as it is.

**Review by nassdonald — APPROVED** [review]

**Pull request merged**